src/Elasticsearch.Net/ElasticLowLevelClient.Snapshot.cs

```diff
@@ -45,17 +45,15 @@ internal LowLevelSnapshotNamespace(ElasticLowLevelClient client): base(client)
 
 		///<summary>POST on /_snapshot/{repository}/_cleanup <para>https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-snapshots.html</para></summary>
 		///<param name = "repository">A repository name</param>
-		///<param name = "body"></param>
 		///<param name = "requestParameters">Request specific configuration such as querystring parameters &amp; request specific connection settings.</param>
-		public TResponse CleanupRepository<TResponse>(string repository, PostData body, CleanupRepositoryRequestParameters requestParameters = null)
-			where TResponse : class, IElasticsearchResponse, new() => DoRequest<TResponse>(POST, Url($"_snapshot/{repository:repository}/_cleanup"), body, RequestParams(requestParameters));
+		public TResponse CleanupRepository<TResponse>(string repository, CleanupRepositoryRequestParameters requestParameters = null)
```

Comment:
I think this is because I removed the empty `"body": {}` from this API

Reply:
Yes, this will surface as a breaking change.

Reply:
Reinstated in https://github.com/elastic/elasticsearch-net/pull/4540/commits/39c595dbd145872efeccaab492f67470bf525331